couchdbkit/ext/django/schema.py

```diff
@@ -75,7 +76,7 @@ def contribute_to_class(self, cls, name):
                 # Ignore any private attributes that Django doesn't care about.
                 # NOTE: We can't modify a dictionary's contents while looping
                 # over it, so we loop over the *original* dictionary instead.
```

Comment:
iterating on `some_dict.keys()` and mutating `some_dict` inside the iteration works in Python 2, but will not work in Python 3. However, in this case `self.meta` is a class object, not a dict, so I think that doesn't work here.